### PR TITLE
Added package override for isteven omni bar

### PR DESCRIPTION
**Pull request by XVincentX**
This PR should add the required overrides for [isteven-angular-omnibar](https://github.com/isteven/angular-omni-bar)

### CODE REVIEW

**Comment by guybedford** [comment]
This repo doesn't seem to exist? https://github.com/angular-omni-bar/isteven-angular-omni-bar?

Note that the name is not used in jspm, as we don't provide a publishing workflow (yet).

**Comment by XVincentX** [comment]
The repo is https://github.com/isteven/angular-omni-bar

File: package-overrides/github/isteven/angular-omni-bar@1.0.1.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "isteven-angular-omni-bar",
+  "main": "isteven-omni-bar.js",
+  "dependencies": {
+      "css": "0.1.10"
+    },  
+  "shim": {
+    "isteven-angular-omni-bar": {
```
Comment (guybedford) [inline package-overrides/github/isteven/angular-omni-bar@1.0.1.json]: This should just be `isteven-moni-bar` to refer to the main entry point.

File: package-overrides/github/isteven/angular-omni-bar@1.0.1.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "isteven-angular-omni-bar",
```
Comment (guybedford) [inline package-overrides/github/isteven/angular-omni-bar@1.0.1.json]: This is not necessary.

File: package-overrides/github/isteven/angular-omni-bar@1.0.1.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "main": "isteven-omni-bar.js",
+  "dependencies": {
+      "css": "0.1.10"
```
Comment (guybedford) [inline package-overrides/github/isteven/angular-omni-bar@1.0.1.json]: It's best to use `^0.1.10` here to get new updates.

**Comment by guybedford** [comment]
Ah right, sorry didn't see that. I've included some further feedback above.

**Comment by XVincentX** [comment]
@guybedford I addressed all your comments and the build is also passing. How about squashing the commits into a single one?

**Comment by guybedford** [comment]
Sure that would be great, thanks.

**Comment by XVincentX** [comment]
@guybedford Done.

**Pull request merged**